components/pedigree/resources/src/main/resources/pedigree/view/studySelectionDialog.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * The UI Element for displaying prompts to the user in a movable/semi-transparent dialogue for study selection
+ *
+ * @class OkCancelDialogue
+ */
+define([], function(){
+  var StudySelectionDialog = Class.create( {
+
+      initialize: function(studies) {
+          var _this = this;
+
+          this.studies = studies;
+
+          var mainDiv = new Element('div', {'class' : "study-selection-dialog"});
+          var container = new Element('div', {'class' : 'xform'});
+
+          mainDiv.insert(container);
+
+          var submitButton = new Element('input', {type : 'button', value : "Submit", 'class' : 'button'});
+          mainDiv.insert(submitButton);
+          submitButton.observe('click', function(event) {
+              selectedStudies = $$('.study-selection-dialog input[type="radio"].study-input');
+              var studyName = "";
+              selectedStudies.each( function(item) {
+                  if (item.checked) {
+                      studyName = item.value;
+                  }
+              });
+              _this.submit(studyName);
+              _this.hide();
+          });
+
+          var cancelButton = new Element('input', {type : 'button', value : "Cancel", 'class' : 'button secondary'});
+          mainDiv.insert(' ').insert(cancelButton);
+          cancelButton.observe('click', function(event) {
+        	  _this.hide();
+          });
+
+          var studiesContainer = new Element('dl');
+          var header = new Element('dt').update('Please select one of the available studies');
+          studiesContainer.insert(header);
+
+          var studiesList = new Element('dd');
+          var noneStudy = new Element('div', {'class' : 'form-template-option'});
+          var input = new Element('input', {'type' : 'radio', 'name' : 'study-selection'});
+          input.checked = true;
+          var label = new Element('label').insert(input).insert(' None');
+          noneStudy.insert(label);
+          noneStudy.insert(new Element('p', {'class' : 'xHint'}).insert('Use the system\'s default form configuration: a generic form covering basic patient information and covering phenotypes related to major anatomic systems.'));
```

Comment:
I think it would be better to make it uniform and add the default "no study" as one of the options passed ot the pedigree via the `PedigreeInterface` script (only if there is at least one other study), so that at least the text above is not hardcoded in pedigree Ui code? I'm sure it is a translation, which can be accessed form velocity, which is an added reason to use velocity to generate this

Reply:
"No study" is not one of the studies, the dialog is a right place to handle the accommodation for the this case to present in UI.

Reply:
...but hard-coding the description that is supposed to come from PhenoTips is not